notify/impl.go

```diff
@@ -1322,6 +1322,38 @@ func (n *Pushover) retry(statusCode int) (bool, error) {
 	return false, nil
 }
 
+type renderer struct {
+	tmpl *template.Template
+	data *template.Data
+
+	err error
+}
+
+func newRenderer(tmpl *template.Template, data *template.Data) *renderer {
+	return &renderer{
+		tmpl: tmpl,
+		data: data,
+	}
+}
+
+func (r *renderer) TextString(s string) (out string) {
```

Comment:
Why doesn't TextString return an `error`? It would make error handling more explicit. In addition for anyone having a linter like [errcheck](https://github.com/kisielk/errcheck) ignored errors would be very quickly detected.

Is it for reducing the error handling noise in the code?

Reply:
Yes, this is to have less verbose code. As you said earlier it can be discussed/addressed in a separate PR since it affects (almost) all receivers. I can rollback the last commit and we could merge this one for now. WDYT?

Reply:
Separate PR sounds good, I will take care of that. I am sorry for the extra work. Let's rollback the last commit and proceed.

Reply:
No problem, I always meant the second commit to be more for discussion than being merged.